bindings/wasm/package.json

```diff
@@ -71,6 +71,8 @@
     "ts-node": "^10.9.1",
     "tsconfig-paths": "^4.1.0",
     "txm": "^8.1.0",
+    "typedoc": "^0.24.6",
+    "typedoc-plugin-markdown": "^3.14.0",
```

Comment:
We should update them some day. But for now I only got it working with these versions 😅